Add note about cluster state diffs (#39847)
elastic/elasticsearch: docs/reference/modules/discovery/publishing.asciidoc

cluster state update within this time then it is considered to have failed and
is removed from the cluster.

Cluster state updates are typically published as diffs to the previous cluster

> state, which helps in saving on network bandwidth as well as speeding up the

Reviewer: ```suggestion
state, which reduces the time and network bandwidth needed to```

> publishing of cluster state updates. For example, when updating the mappings

Reviewer: ```suggestion
publish a cluster state update. For example, when updating the mappings
```
